app/connectors/AmlsConnector.scala

```diff
@@ -240,6 +207,18 @@ class AmlsConnector @Inject()(val httpPost: WSHttp,
     httpPost.POSTString[HttpResponse](postUrl, paymentId)
   }
 
+  def savePayment(paymentId: String, amlsRefNo: String, safeId: String, accountTypeId: (String, String))
+                 (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
+
+    val (accountType, accountId) = accountTypeId
+    val postUrl = s"$paymentUrl/$accountType/$accountId/$amlsRefNo/$safeId"
+
+    Logger.debug(s"[AmlsConnector][savePayment]: Request to $postUrl with paymentId $paymentId")
+
+    httpPost.POSTString[HttpResponse](postUrl, paymentId)
+  }
+
+  @deprecated("to be removed after new auth changes implemented")
   def getPaymentByPaymentReference(paymentReference: String)
                                   (implicit hc: HeaderCarrier, ec: ExecutionContext, ac: AuthContext): Future[Option[Payment]] = {
     //TODO - deprecated by AuthAction.accountTypeAndId after new auth changes
```

Comment:
Is there a need for this and @deprecated?

Reply:
removed